src/ansys/systemcoupling/core/client/grpc_client.py

```diff
@@ -153,7 +156,7 @@ def start_container_and_connect(
         LOG.debug("...started")
         self._connect(_LOCALHOST_IP, port)
 
-    def start_pim_and_connect(self, version: str = None):
+    def start_pim_and_connect(self, start_output: bool, version: str = None):
```

Comment:
For consistency with `start_and_connect`, `start_output` should probably be optional and defaulted to `False` here.

Reply:
Thanks for catching this @iboyd-ansys, its resolved now.